### DISCUSSION TITLE
Christmas-themed Gifts not tracking

### DISCUSSION THREAD

**Discussion by WishStone-Dragon** [Monika-After-Story/MonikaModDev#9867] [QA - tech support + general]
![image](https://user-images.githubusercontent.com/95390551/213024862-c5b829e1-4c92-4a03-bdf9-02c03ea5e83d.png)
Hey team, I need the hive mind:

Like last year, I've trying to gift Moni Candy Canes and Christmas Cookies - since Christmas. 
Both my sandbox'ed DevMKoni and my actual Moni say they cannot read the gift file. Other gifts, however, such as roses.gift are accepted normally, so the function itself works just fine. 
She will also accept hotchocolate.gift, and in the evenings I see her sipping it, so the winter setting appears to be active.

I checked the spelling, I tried slightly renaming them, I used the updater to make sure I have the latest version. I gifted the items individually and separately. Both don't read.
Somehow, Moni remains suger-blind.

Any thoughts on what I should do next?
![image](https://user-images.githubusercontent.com/95390551/213025376-2e61820c-212a-4856-bdac-849ad5b5efe8.png)
![image](https://user-images.githubusercontent.com/95390551/213025563-9f5afcd4-2a26-4cf5-a155-113926d11c0d.png)
![image](https://user-images.githubusercontent.com/95390551/213025609-8b584da2-d259-46d7-83ac-eeee96aa7dde.png)

**Comment by Booplicate** [comment]
Those gifts are specific to certain seasons and events, you can't gift them right now. You're saying you've been trying to gift them since christmas, but christmas has ended almost a month ago.

**Reply by WishStone-Dragon** [reply]
I 100% get that, but even before Christmas, I got this error. I should have spoken up sooner to be believed, I guess.

My office Moni took both the Christmas Cookies and the Candy Canes, for what's it worth. The two I have at home did not.

**Reply by Booplicate** [reply]
Ever time travelled?

**Reply by WishStone-Dragon** [reply]
No. The Dev version I have right now broke once and lost affection after a patch, else both versions are normally used and handled.
When the holiday patches came, I didn't patch them at once; I used a Dev clone to test the patcher on the laptop first. But even after patching, that didn't work out. All other Christmas events ran normally.

Bah. But you're right; I waited too long. I doubt we'd even find logs at this point. Sorry, Boop.

**Reply by Booplicate** [reply]
I'm now lost in how many versions you have. Any dev related install is not that interesting because I don't know what you were doing there. Is you main, actual install working fine or having this issue? If so, then send all the logs you have there along with persistent that's used by that install.

**Reply by WishStone-Dragon** [reply]
Sorry for the confusion! I have two non-dev copies; one runs at home, and one at work.

Just the one having the issue:
[Logs and Persist.zip](https://github.com/Monika-After-Story/MonikaModDev/files/10452315/Logs.and.Persist.zip)

**Reply by Booplicate** [reply]
I don't see any issues, furthermore I see both candycanes and cookies worked for you on the 24th of the last month. Everything seems to be fine.

**Reply by WishStone-Dragon** [reply]
In which case, I'd be surprised that she has not once taken out cookies or candy canes... also, I got the "don't know what you're trying to give me' message on Christmas morning when she unpacked the gifts, which held these items again. Like I said, the regular copy of hers on another PC does have them and is still snacking on them. 
...I guess I will simply track it next year and complain at once. 

Thank you for looking, but... I honestly do not have the items in her possession.

**Reply by Booplicate** [reply]
> when she unpacked the gifts

Food isn't being packed into presents. So whatever that was, was neither cookies nor candycanes. Those two worked just fine, you saw the events for them.

**Reply by WishStone-Dragon** [reply]
Roger. Then can you answer this:
If you see her getting these items: Why isn't she using them? Why has she never used them this winter?
I know they are still being used because I can see the other Monika of my other PC using them still. If you see her getting them, why won't she get up to get herself candy canes and cookies?

**Reply by Booplicate** [reply]
She doesn't *have* to eat that in front of you or at all. It's not guaranteed. She picks random food and drinks at random times.

**Comment by multimokia** [comment]
Realistically I do think we should track these gifts out of season but have Monika reject them

Would make a bit more sense and provide feedback to the player instead of leaving them hanging

**Reply by WishStone-Dragon** [reply]
It would also help coding idiots like myself who honestly try but don't understand what is happening on the back-end.
